README.md

```diff
@@ -445,6 +446,74 @@ At this point, we can start the web application as follows:
 
 Please consult the `help` CLI command for more insights on the various options.
 
+
+# Configuration
+
+To ease life of users, Trento can be run with a config file in replacement of command-line arguments.
+
+## Locations
+Configuration, if not otherwise specified by the `--config=/path/to/config.yaml` option, would be searched in following locations:
+
+Note that order represents priority
+
+- `/etc/trento/` <-- first location looked
+- `/usr/etc/trento/` <-- fallback here if config not found in previous location
+- `~` aka user's home <-- fallback here
+
+## Formats
+
+Supported formats are the ones suppoerted by [Viper](https://github.com/spf13/viper), and we encourage `yaml` usage.
```

Comment:
Typo here. Also the word "supported" is repeated.

Besides that, did you actually try using some other format? I remember experiencing issues with that in the past, so we ended up locking viper to only support `yaml` and call it done.

Reply:
How can I rephrase it?

`Supported formats are the ones that Viper supports`, good?

And, no, I actually didn't try other formats. Shall we communicate to only use `yaml` and remove reference to supported formats in VIper?

Reply:
Know what, gonna stick on `yaml`.

I tried out `toml` and we'd have to consider namespacing by default, for configuration groups.

```
# TOML

[mTLS]
enable-mtls = false
cert = "./test/certs/client-cert.pem"
key = "./test/certs/client-key.pem"
ca = "./test/certs/ca-cert.pem"
```

This becomes accessible by `viper.GetBool("mTLS.enable-mtls")`

In yaml it would be
```
mTLS:
    enable-mtls: false
    cert: ./test/certs/client-cert.pem
    key: ./test/certs/client-key.pem
    ca: ./test/certs/ca-cert.pem
```

and access the same way  by `viper.GetBool("mTLS.enable-mtls")`.

I do not dislike this namespacing, yet I'd keep it simple and stick on `yaml`.

Deal?

Reply:
Either way works for me. Namespacing is kinda cool, but it also makes very ugly CLI flags: e.g. `--mtls.enable-mtls`. We would need to review _all of them_ and rename _a lot_ to make sure there are no funky ones.

Reply:
indeed. I wouldn't do it NAO :joy: 
So simple yaml shou;d be enough